htlcswitch/link.go

```diff
@@ -3320,6 +3321,14 @@ func (l *channelLink) processRemoteAdds(fwdPkg *channeldb.FwdPkg,
 					PaymentHash: pd.RHash,
 				}
 
+				if fwdInfo.NextBlinding != nil {
+					addMsg.BlindingPoint = tlv.SomeRecordT(
```

Comment:
Independent of the comment above, can make a `someBlindingPoint` (or w/e) func to hide some of these details.